nemo_text_processing/g2p/modules.py

```diff
@@ -377,6 +379,16 @@ def _parse_phoneme_dict(phoneme_dict: Union[str, pathlib.Path]):
             phoneme_dict_obj = phoneme_dict
         return phoneme_dict_obj
 
+    def setup_heteronym_model(
```

Comment:
Still need to discuss how we want to set this up (across G2P classes or per-instance)... Just putting a note here so we don't forget!